Yona/.rswiftignore

```diff
@@ -0,0 +1,4 @@
+# Fabric and Crashlytics files
+fabric.apikey
+fabric.buildsecret
```

Comment:
This file and the next one shouldn't exist and should be removed here.

Reply:
Removed